Rozdział 8/Hangman2.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Hangman2!</title>
+</head>
+<body>
+    <script>
+    
+    var words = ["javascript", "monkey", "amazing", "pancake", "tractor", "blablabla"];
```

Comment:
`""` -> `''` - zazwyczaj używa się pojedynczych, ale to zawsze zależy od konwencji w projekcie - to tak bardziej FYI